tests/network/vmi_infosource.go

```diff
@@ -75,6 +78,15 @@ var _ = SIGDescribe("Infosource", func() {
 		secondaryNetwork2 := libvmi.MultusNetwork(secondaryInterface2Name, nadName)
 
 		BeforeEach(func() {
+			fgDisabled := !checks.HasFeature(virtconfig.DynamicPodInterfaceNamingGate)
+			if fgDisabled {
+				config.EnableFeatureGate(virtconfig.DynamicPodInterfaceNamingGate)
+			}
+			DeferCleanup(func() {
+				if fgDisabled {
+					config.DisableFeatureGate(virtconfig.DynamicPodInterfaceNamingGate)
+				}
+			})
```

Comment:
I want to propose that we don't enable the feature gates unless needed to not pollute env. So can we rather enable this feature gate for the test suite and mark the tests with label so we are able to focus/skip?

Reply:
We should enable the feature only on a specific test, preferably one that is already serial.

Reply:
Do I have to mark the test suite as serial ?